cmd/atlas/internal/client/driver.go

```diff
@@ -0,0 +1,109 @@
+package client
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"ariga.io/atlas/sql/mysql"
+	"ariga.io/atlas/sql/postgres"
+	"ariga.io/atlas/sql/schema"
+	"ariga.io/atlas/sql/sqlite"
+)
+
+type (
+	// Driver provides to diff, execute and inspect against a Data Source.
+	Driver interface {
+		schema.Differ
+		schema.Execer
+		schema.Inspector
+	}
```

Comment:
I don't think this interface is needed, a type that embeds these interfaces as you did below is fine.